packages/nextra-theme-docs/package.json

```diff
@@ -48,7 +48,7 @@
     "@popperjs/core": "^2.11.6",
     "@reach/skip-nav": "^0.17.0",
     "clsx": "^1.2.1",
-    "flexsearch": "^0.7.21",
+    "flexsearch": "0.7.21",
```

Comment:
> By the way, please note that I had to pin "flexsearch": "0.7.21" on a clean rebuild of the pnpm-lock.yaml to get around a failure in pnpm install.

@robbear would you mind share the error you've been getting? Since I tried on my machine everything works well.

Reply:
I was seeing this when I:

1. Start from a fresh clone (specifically, no `node_module` contents)
2. Remove `pnpm-lock.yaml` so as to regenerate it
3. `pnpm install`

![image](https://user-images.githubusercontent.com/1823198/205676450-b8eb0c83-f91a-4674-a19b-3f540b6e3617.png)

Pinning `flexsearch` to `0.7.21` resolves it.